src/ida_lunde_naalsund_ex/ex05/bounded_sim.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Ida Lunde Naalsund"
+__email__ = "idna@nmbu.no"
+
+from walker_sim import Walker, Simulation
+import random
+
+
+class BoundedWalker(Walker):
+    def __init__(self, start, home, left_limit, right_limit):
+        """
+        Initialise the walker
+
+        Arguments
+        ---------
+        start : int
+            The walker's initial position
+        home : int
+            The walk ends when the walker reaches home
+        left_limit : int
+            The left boundary of walker movement
+        right_limit : int
+            The right boundary  of walker movement
+        """
+        self.start = start
+        self.home = home
```

Comment:
Should not be reassigned. This is "the job" of the `super().__init__()`. Which should be called first, btw.